datastore/datastore.go

```diff
@@ -426,22 +427,27 @@ func (c *Client) GetMulti(ctx context.Context, keys []*Key, dst interface{}) (er
 		}
 	}
 
-	return c.get(ctx, keys, dst, opts)
+	// TODO: Use transaction ID returned by get
+	_, err = c.get(ctx, keys, dst, opts)
+	if err != nil {
+		return err
+	}
+	return nil
```

Comment:
Here's another example of where we could use `return err`.

Reply:
Sorry about this. Fixed!